// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";


message t_dlu_stats {
   required uint32 ue_index = 1;
   required uint32 rb_id = 2;
   required uint32 sdu_count = 4;
   required uint32 total_win = 7;
   required uint32 min_win = 8;
   required uint32 max_win = 9;
   required uint32 total_delay = 10;
   required uint32 min_delay = 11;
   required uint32 max_delay = 12;
   required uint32 delay_count = 13;
   required uint32 total_queue_B = 14;
   required uint32 min_queue_B = 15;
   required uint32 max_queue_B = 16;
   required uint32 total_queue_pkt = 17;
   required uint32 min_queue_pkt = 18;
   required uint32 max_queue_pkt = 19;
   required uint32 queue_count = 20;
}

message dl_south_stats {
   required uint64 timestamp = 1;
   repeated t_dlu_stats stats = 2;
}




